src/assets/theme/searchbar.ts

```diff
@@ -16,7 +16,7 @@ export interface SearchbarTheme {
 
 export const searchbar: SearchbarTheme = {
   mobileClosedWidth: `${mobileWidth}px`,
-  mobileClosedPadding: '0.75rem 0.5rem 1.5rem',
+  mobileClosedPadding: '0.75rem 0.5rem 1rem',
```

Comment:
lets keep this `1.5rem` to keep it close to [the mocks](https://www.figma.com/file/JGqemgZwlG9e5DQbZpdKgy/Redesign?node-id=1476%3A143)